### DISCUSSION TITLE
doris0.15.0 be 在查询数据中挂掉

### DISCUSSION THREAD

**Discussion by sanyiss** [apache/doris#7537] [A - General / Q&A]
start time: Wed Dec 29 17:17:05 CST 2021
*** Aborted at 1640779186 (unix time) try "date -d @1640779186" if you are using GNU date ***
PC: @          0x15da14b doris::SortedRunMerger::heapify()
*** SIGSEGV (@0xe38) received by PID 12488 (TID 0x7f4c7a089700) from PID 3640; stack trace: ***
    @          0x214c712 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7f4d28a2e100 (unknown)
    @          0x15da14b doris::SortedRunMerger::heapify()
    @          0x15da713 doris::SortedRunMerger::get_next()
    @          0x15da860 doris::ChildSortedRunMerger::get_batch()
    @          0x15b4a3f std::_Function_handler<>::_M_invoke()
    @          0x15db6ae doris::SortedRunMerger::ParallelBatchedRowSupplier::process_sorted_run_task()
    @          0x3907400 execute_native_thread_routine
    @     0x7f4d28a26dc5 start_thread
    @     0x7f4d2845221d __clone
    @                0x0 (unknown)

**Comment by sanyiss** [comment]
SQL：select * from  tableName  as a join (select PK_ID from tableName  where 1=1 order by PK_ID asc limit 10000000,100000 ) as b on a.PK_ID = b.PK_ID

**Comment by cooldwind** [comment]
dmesg -T,看下，是不是有内存超过限制。PK_ID 是string类型吗，string类型换成varchar。

**Reply by Bugman-Philip** [reply]
遇到相同的问题，但是提高exec_mem_limit参数， 依旧报内存不足；比如说设置exec_mem_limit为8G，提示为使用了8.1G,报错；
将exec_mem_limit设置为16G,提示为使用了16.2G，报错；

**Reply by dataalive** [reply]
可以走下shuffle join尝试下。join [shuffle]类似这样的用法。

**Comment by hlh5** [comment]
同个问题， doris 0.15.0 下 uniq 表，查询string 字段报错
`hometown` text  NULL COMMENT "家乡"
mysql> select count(*) from dim_ht_imdb_user where hometown is null;
ERROR 1064 (HY000): RpcException, msg: rpc failed, host: 10.10.2.99

**Reply by dataalive** [reply]
可以升级到0.15.3后或者最新的1.0.0-rc02来试试。之前的版本有string类型的bug fix。
